forem/forem
# Implement Unified Embed for Vimeo URLs

## What type of PR is this? (check all applicable)


## Description
This PR implements the [unified embed tag](https://github.com/forem/forem/issues/15099) for Vimeo.

## Related Tickets & Documents
closes https://github.com/forem/forem/issues/15584

## QA Instructions, Screenshots, Recordings

Example Vimeo URLs:
https://player.vimeo.com/video/652446985?h=a68f6ed1f5
https://vimeo.com/ondemand/withchude/647355334
https://vimeo.com/636725488

As a user, try creating Vimeo liquid tags using this format: `{% embed <vimeo-url> %}`. Please use the example Vimeo links listed above. The Vimeo videos should render properly. Also create vimeo liquid tags using the old method: `{% vimeo <vimeo-url> %}`. This should also work, as I am leaving the old Vimeo implementation in place for now.

### UI accessibility concerns?

None

## Added/updated tests?

      have not been included_

## [Forem core team only] How will this change be communicated?
Considering the project as a whole, I shall update the Liquid Tag documentation at the end. Since I am not changing the current implementation, the present documentation is still valid. Plus, I will avoid updating the docs piecemeal, and risk confusing users.
